com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -53,47 +53,69 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
 
 **Minimum Depth** set to 2, **Maximum Depth** set to 2: indirect lighting only (1 bounce)
 
-## Materials parameterization
+## Setting path tracing parameters for Materials
+
+
+Path tracing in HDRP makes your scene appear more realistic. To do this, path tracing implements more precise light transport simulations than rasterization. The result is more realistic effects, especially for:
 
-Some light phenomena can be expressed more naturally in path tracing than in rasterization, for instance:
 
 - Light refraction in transmissive objects.
 - Light absorption in transmissive objects.
 - Subsurface scattering.
 
-Rasterization uses various methods to approximate complex lighting effects, with each of those methods relying on dedicated Material parameters. Path tracing computes those same effects all at once, without the need for as many parameters.
+Rasterization uses separate methods to approximate lighting effects, which require multiple Material parameters. Path tracing computes all lighting effects and how light interacts with Materials at the same time, which means Materials in your scene need fewer Material parameters.
+
+Some parameters have no effect when you use path tracing, and path tracing also changes how a Lit Material’s refraction model behaves. For more information, see [Refraction models](#refraction-models).
+
+## How to enable path tracing on a Material
+
```

Comment:
This is incorrect, the re-wording here is wrong. There is no "path tracing" option on a Material.
We used exactly the same Material with or without PathTracer, only the interpretation of the parameters is changing.

And 
>Path tracing only works on [Lit materials](Lit-Shader.md).
this is incorrect, path tracing works for all Materials.

I guess the initial document wasn't clear enough. We want to highlight that "refrcation model" in the lit shader behave differently, but we still support other Material. And ultimatly, the refraction model could be extent to other shader in the future like "stacklit". For the sake of this documentation, I think we can simply focus on lit's refraction model and say that the interpretation of this parameter on the lit shader is different when path tracing is enabled, and then we explain what is different.

Reply:
currenlty this is what you write in:

"Path tracing changes the way refraction models on a Lit Material behave."

so I guess we just need to removed the section: ## How to enable path tracing on a Material - guess this section is causing confusion.